src/SurfacePlot/index.ts

```diff
@@ -0,0 +1,8 @@
+// This file exports the default export of the SurfacePlotClientOnly module
+// and the SurfacePlotProps type from the SurfacePlot.types module.
```

Comment:
Don't put sentences on to multiple lines like this. It might look good with your particular width of screen but not for everybody. Just put on one line and let people turn on word wrap in their IDE if they so desire.